src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -180,6 +202,17 @@ def _describe(self, catalog: str, database: str, table: str) -> Table | None:
             logger.error(f"Couldn't fetch information for table {full_name} : {e}")
             return None
 
+    def unset_upgraded_to(self, database: str | None = None, name: str | None = None, *, deletemanaged: bool = False):
```

Comment:
Incorrect logic: an assessment job could be run again after we start the table migration and upgraded_to field will get overwritten, loosing migration data, making migration logic incorrect. Table properties is the source of truth.

Remove this method.

Reply:
What if I run the command for a single table?
Do I have to rerun the assessment?

Reply:
Removed